pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -84,15 +97,38 @@ func (n *VMNetworkConfigurator) SetupPodNetworkPhase1(pid int) error {
 	return nil
 }
 
-func (n *VMNetworkConfigurator) SetupPodNetworkPhase2(domain *api.Domain) error {
-	nics, err := n.getNICs()
+func NewNetworkingSpecGenerator(vmi *v1.VirtualMachineInstance, cacheFactory cache.InterfaceCacheFactory, domain api.Domain) NetworkingSpecGenerator {
+	networkDriver := &netdriver.NetworkUtilsHandler{}
+	return NetworkingSpecGenerator{
+		vmi:          vmi,
+		handler:      networkDriver,
+		cacheFactory: cacheFactory,
```

Comment:
Non of this are used by NetworkingSpecGenerator we can remove them, they are only used by the NicGenerator